climpred/bootstrap.py

```diff
@@ -160,20 +257,25 @@ def _bootstrap_func(
         sig_level: bootstrapped significance levels with
                    dimensions of ds and len(sig) if sig is list
     """
+    if not callable(func):
```

Comment:
This would be good to add to `checks.py` because it can be used in some metrics too.